### ISSUE TITLE
Failing to get game settings

### ISSUE THREAD

**Issue by hcribeiro** [death-save/pin-cushion#30]
Getting: 

```js
Error: This is not a registered game setting
    at ClientSettings.get (foundry.js:3436)
    at pin-cushion.js:245
    at Function._call (foundry.js:2496)
    at Function.callAll (foundry.js:2456)
    at Note._onHoverOut (foundry.js:11839)
    at Note._onHoverOut (foundry.js:42638)
    at MouseInteractionManager.callback (foundry.js:36392)
    at MouseInteractionManager._handleMouseOut (foundry.js:36531)
    at ControlIcon.a.emit (index.js:181)
    at r.dispatchEvent (InteractionManager.ts:1106)
```

Going into that file, it looks like it's failing to fetch those variables (`showJournalPreview` and `previewDelay`). Running that command separately confirms that those variables aren't getting set properly. I saw that #15 and #11 had the same issue, so this seems intermittent.

**Comment by eclarke12** [comment]
It might be a race condition I missed somewhere. I'll take another look next time I open the mod for updates

**Comment by eclarke12** [comment]
Will be resolved in v1.2
